README.md

```diff
@@ -27,30 +27,31 @@ It's mighty because it:
 
 It's easy to get started.
 
-First, decide how you want to use stylelint:
+First, decide if you want to use stylelint:
 
 -   [on the command line](docs/user-guide/cli.md)
 -   [in your text editor](docs/user-guide/complementary-tools.md#editor-plugins), for example in VS Code
--   [in for your build tool](docs/user-guide/complementary-tools.md#build-tool-plugins), for example in webpack
+-   [in for your build tool](docs/user-guide/complementary-tools.md#build-tool-plugins), for example in Webpack
```

Comment:
`webpack` is like stylelint, all spelled with lowercase `webpack`

See https://webpack.js.org/branding/#the-name